merlin/models/tf/blocks/dlrm.py

```diff
@@ -134,5 +124,45 @@ def DLRMBlock(
     return top_block_outputs
 
 
+def _get_embeddings(embedding_dim, embedding_options, bottom_block, cat_schema):
+    if embedding_dim is None:
+        raise ValueError("The embedding_dim is required")
+
+    if embedding_options is not None:
+        embedding_options.embedding_dim_default = embedding_dim
+        embedding_options.infer_embedding_sizes = False
+    else:
+        embedding_options = EmbeddingOptions(embedding_dim_default=embedding_dim)
+
+    if (
+        embedding_dim is not None
+        and bottom_block is not None
+        and embedding_dim != bottom_block.layers[-1].units
+    ):
+        raise ValueError(
+            f"The embedding_dim ({embedding_dim}) needs to match the "
+            "last layer of bottom MLP ({bottom_block.layers[-1].units}) "
+        )
+
+    embeddings_kwargs = dict(
+        sequence_combiner=embedding_options.combiner,
+        embedding_dims=embedding_options.embedding_dims,
+        embedding_dim_default=embedding_options.embedding_dim_default,
+        infer_embedding_sizes=embedding_options.infer_embedding_sizes,
+        infer_embedding_sizes_multiplier=embedding_options.infer_embedding_sizes_multiplier,
+    )
+    embeddings_kwargs["infer_embeddings_ensure_dim_multiple_of_8"] = (
+        embedding_options.infer_embeddings_ensure_dim_multiple_of_8,
+    )
+    embeddings = SequentialBlock(
```

Comment:
Why do we wrap this in a `SequentialBlock`?

Reply:
This is because the `EmbeddingTable` doesn't support list-like (tuple) values in the inputs. Alternatively, we could implement support for this in that layer.